src/os.h

```diff
@@ -34,6 +34,7 @@
 
 #if defined(STRESSAPPTEST_CPU_X86_64) || defined(STRESSAPPTEST_CPU_I686)
 #include <immintrin.h>
+#include <x86intrin.h>
```

Comment:
Could you help also consider the fix on other arches? Like aarch64 because our test on arm also fails due to this issue. Thanks.